index.Rmd

```diff
@@ -34,6 +35,29 @@ Scripting with *R* to handle spatial problems requires a core set of basic *R* s
 **Note** that this tutorial uses the basic programming structure with lists, strings, and more. For a refresher on these elements click [here](https://geoscripting-wur.github.io/RPythonBasics/).
 ```
 
+## Package installation and creating new directories 
+To code efficiently, we use packages that contain ready-made features for specific applications. The relevant packages must first be installed before using their features. In this tutorial, we will be using the *terra* and *sf* packages. We can therefore go ahead and install them now. 
+
+```{r}
+# Check for the terra and sf packages and install if missing
+if(!"terra" %in% installed.packages()){install.packages("terra")}
+if(!"sf" %in% installed.packages()){install.packages("sf")}
+```
+
+We first check if the package has already been installed, to avoid unnecessary steps. If the package is missing, then we install it. 
+
+```{r}
+# Load the terra and sf packages 
+library(terra)
+library(sf)
+```
+
+Similarly, we can create new directories for storing out data using the functions `dir.exists` and `dir.create`. Again, we check if these directories are missing before creating them, to avoid redundancy. 
+
+```{r}
+if(!dir.exists("data")){dir.create("data")}
+```
+
```

Comment:
Here is a short section on package installation and how to create directories (I felt that we might as well add the second part here since the code is pretty similar). Let me know if you see some things missing in this section.